task_21/toma.lobanova/src/test/java/com/hillel/tasks/train/TrainTests.java

```diff
@@ -0,0 +1,50 @@
+package com.hillel.tasks.train;
+
+import org.junit.Test;
+
+public class TrainTests {
```

Comment:
In the real world you should test all options, not only when a method throws an exception